terraform/serverless/05_app_engine.tf

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_app_engine_standard_app_version" "ratingservice_v1" {
+  version_id = "v1"
+  service    = "ratingservice"
+  runtime    = "python38"
+
+  entrypoint {
+    shell = "uwsgi --http-socket :8080 --wsgi-file main.py --callable app --master --processes 1 --threads 2"
+  }
+
+  deployment {
+    zip {
+      source_url = "https://storage.googleapis.com/${var.bucket_name}/ratingservice_code:latest.zip"
```

Comment:
instead of "latest" it should be a variable that holds the value of the release. It's value should be "latest" by default and be updated upon releasing to corresponding version of the release.

Reply:
I added it.